elements/pfe-tabs/demo/demo.css

```diff
@@ -57,18 +57,34 @@ section.pfe-l-grid {
 .example>h2 {
     text-align: right;
 }
+section>h2 {
+    margin-top: 2em;
+}
 
 .subheading {
   margin: 0;
   font-size: 18px;
   font-weight: bold;
   font-family: "Arial";
 }
-
+.inner {
+   margin: auto;
+   max-width:1140px;
+   padding: 50px 0;
+}
+.light-background {
+    background-color: rgb(252, 250, 244);
+    --theme: light;
+}
 .dark-background {
-    background-color: #151515;
+    background-color: #524848;
```

Comment:
these colors are now intentionally weird shades so you can see the colors of the tab headers & panels